Flaky periodic-kueue-test-large-scale-scheduling-perf-release

<!--
Please use this template for reporting flaky tests.
Links to specific failures in Prow are appreciated.
-->

**Which test is flaking?**:
Flaky Performance Test

**Link to failed CI job or steps to reproduce locally**:
https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/periodic-kueue-test-large-scale-scheduling-perf-release-0-18/2061287550114336768

**Failure message or logs**:
```
{Failed  === RUN   TestScalability/WorkloadClasses/large
    checker_test.go:112: Average wait for admission 123077ms is more then expected 119000ms
--- FAIL: TestScalability/WorkloadClasses/large (0.00s)
}
```

**Anything else we need to know?**:

--- a/test/performance/scheduler/configs/large-scale/rangespec.yaml
+++ b/test/performance/scheduler/configs/large-scale/rangespec.yaml
@@ -21,6 +21,6 @@ clusterQueueClassesMinUsage:
 
 wlClassesMaxAvgTimeToAdmissionMs:
   # mean×1.2 rounded up to nearest 1000ms
-  large: 119_000   # mean=98803ms
+  large: 128_000   # mean=105604ms
   medium: 308_000  # mean=256413ms
   small: 841_000   # mean=700379ms

